src/EbtcZapRouter.sol

```diff
@@ -64,13 +67,21 @@ contract EbtcZapRouter is IEbtcZapRouter {
         PositionManagerPermit memory _positionManagerPermit
     ) external returns (bytes32 cdpId) {
         uint256 _collVal = _transferInitialStETHFromCaller(_stEthBalance);
-        return _openCdpWithPermit(
+
+        require(
```

Comment:
This `require` seems not necessary. 

Please check https://docs.lido.fi/guides/lido-tokens-integration-guide/#1-2-wei-corner-case